napari/components/layerlist.py

```diff
@@ -123,6 +124,36 @@ def _extent_world(self) -> np.ndarray:
         """
         return self._get_extent_world([layer.extent for layer in self])
 
+    def _get_min_and_max(self, mins_list, maxes_list):
+
+        # Reverse dimensions since it is the last dimensions that are
+        # displayed.
+        mins_list = [mins[::-1] for mins in mins_list]
+        maxes_list = [maxes[::-1] for maxes in maxes_list]
+
+        with warnings.catch_warnings():
+            # Taking the nanmin and nanmax of an axis of all nan
+            # raises a warning and returns nan for that axis
+            # as we have do an explict nan_to_num below this
```

Comment:
```suggestion
            # as we have do an explicit nan_to_num below this
```

Not your typo, but good to fix anyway.